### ISSUE TITLE
JSF:Jumping from a cell on a row to the same cell on the next row by using up and down arrows is not possible.

### ISSUE THREAD

**Issue by YurinM** [TeamDev-Ltd/OpenFaces#119]

**Comment by YurinM** [comment]
Verified with: openfaces v2.1, browsers: IE11 - Can`t reproduce.

**Comment by YogeshTieto** [comment]
Steps to reproduce:
1.  Create a table having keyboard navigation with 4 columns ,all of them should be input field (EmpFirstName, EmpLastName, EmpAdd, EmpSalary)
2. No. of rows 5 
3.  Select second row and focus should be in EmpLastName column.
4.  Now press Keyboard down arrow key
5. Third row should get selected and focus should move to third row EmpLastName column.
6. Focus move is not happening.
